Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs

```diff
@@ -42,7 +42,12 @@ protected override void OnCreate(Bundle bundle)
 			if (!Debugger.IsAttached)
 				Insights.Initialize(App.InsightsApiKey, ApplicationContext);
 
+#if TEST_EXPERIMENTAL_RENDERERS
+			Forms.SetFlags("FastRenderers_Experimental");
```

Comment:
I thought that when we discussed it we decided that our stuff doesn't need namespaces, but we would recommend that third parties using this should prefix with namespaces.